src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,97 @@
-export const Pagination = () => {};
+import React, { FunctionComponent } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import classNames from 'classnames';
+
+type Props = {
+  itemsOnPage: number[]
+  currentPage: number;
+  totalPages: number[];
+  onPageChange: (event: React.MouseEvent<HTMLElement>) => void;
+};
+
+export const Pagination: FunctionComponent<Props> = ({
+  itemsOnPage,
+  currentPage,
+  totalPages,
+  onPageChange,
+}) => {
+  // const isActivePage = (num: number): boolean => num === currentPage;
+  const isPrevPage = currentPage === 1;
+  const isNextPage = currentPage === totalPages.length;
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={classNames(
+            'page-item', {
+              disabled: isPrevPage,
+            },
+          )}
+        >
+          <a
+            data-cy="prevLink"
+            id="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={isPrevPage}
+            onClick={onPageChange}
+          >
+            «
+          </a>
+        </li>
+
+        {totalPages.map((pageNum) => (
+          <li
+            className={classNames(
+              'page-item',
+              { active: pageNum === currentPage },
+            )}
+            key={uuidv4()}
```

Comment:
you can't call function for generating uuid inside the map callback
in this way you'll have new key on each rendering, it's not help to optimize rendering
you should generate key before rendering list